src/router/index.js

```diff
@@ -1,43 +1,43 @@
 'use strict';
 
 const express = require('express');
-const db = require('./db');
+const db = require('./../db');
+const productsRoutes = require('./products');
 
 const router = express.Router();
 
+router.get('/', productsRoutes.allProducts);
```

Comment:
I like you have put all your code in the function for your route ie. allProducts. And that you have also separated the code and placed in a different file, productsRoutes. Cleaner code and easy to read. Good work!